nva-datamodel-java/src/main/java/no/unit/nva/model/AdditionalIdentifier.java

```diff
@@ -1,78 +1,8 @@
 package no.unit.nva.model;
 
-import static no.unit.nva.DatamodelConfig.dataModelObjectMapper;
-import static nva.commons.core.attempt.Try.attempt;
-import com.fasterxml.jackson.annotation.JsonAlias;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.annotation.JsonTypeInfo;
-import java.util.Objects;
-import nva.commons.core.JacocoGenerated;
 
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-public class AdditionalIdentifier {
+public record AdditionalIdentifier(String sourceName, String value) implements AdditionalIdentifierBase {
 
-    // TODO: Remove alias when data has been migrated
-    @Deprecated
-    @JsonAlias("source")
-    @JsonProperty("sourceName")
-    private String sourceName;
-    @JsonProperty("value")
-    private String value;
+    static final String TYPE = "AdditionalIdentifier";
```

Comment:
Jeg kunne også se for meg hjelpemdetoder som setter strenger med faster verdier for oss:

```
        AdditionalIdentifier.fromScopus(randomString());
        AdditionalIdentifier.fromCristin(randomString());
        AdditionalIdentifier.fromBrage(randomString(), randomString())
```

Reply:
hmm I agree that its convenient and could help making more consistant data, but not sure where to add it. suggest adding it when we need it.

Reply:
added fromBrage and fromCristin helpers to SourceName.

`new HandleIdentifier(SourceName.fromBrage("20754.0.0.0"), randomUri())`

and made singe parameter SourceName constructor package private